conformance/lang/values-types-and-variables/boolean-literal.balt

```diff
@@ -0,0 +1,161 @@
+Test-Case: output
+Description: Test boolean type descriptor.
+Labels: boolean-type-descriptor, boolean-literal
+
+public function main() {
+    boolean x = true;
+    io:println(x); // @output true
+
+    x = false;
+    io:println(x); // @output false
+}
+
+Test-Case: output
+Description: Test boolean literals with function calls.
+Labels: function-type-descriptor, boolean-type-descriptor, boolean-literal
```

Comment:
This should be a `function-defn`, not a `function-type-descriptor`.

Reply:
It would be better if we can order the labels alphabetically for the maintainability.